src/angular-drag-drop.js

```diff
@@ -1,6 +1,14 @@
-var Angular = require('angular');
+var Angular;
 
-module.exports = 'filearts.dragDrop';
+if (typeof angular === 'object') {
```

Comment:
Can you comment on why these changes were necessary?

The build step `npm run build` will transform this file into a UMD-compatible module.